Sources/SwiftCliCore/StitchLibrary.swift

```diff
@@ -1,9 +1,26 @@
 import Foundation
 
-let allowedStitches = ["k1", "p1", "ssk", "k2tog", "yo", "m1", "\n"]
 let stitchPrinting = ["k1": " ",
                       "p1": "-",
                       "ssk": "\\",
                       "k2tog": "/",
                       "yo": "o",
                       "m1": "m" ]
+
+let decreases = ["ssk", "k2tog"]
+let increases = ["m1", "yo"]
+
+struct stitchInfo {
+    var name: String
+    var incDecValue: Int = 0
+    var symbol: String
+}
+
+let allowedStitchesInfo = [stitchInfo(name: "k1", incDecValue: 0, symbol: " "),
+                           stitchInfo(name: "p1", incDecValue: 0, symbol: "-"),
+                           stitchInfo(name: "ssk", incDecValue: -1, symbol: "\\"),
+                           stitchInfo(name: "k2tog", incDecValue: -1, symbol: "/"),
+                           stitchInfo(name: "yo", incDecValue: 1, symbol: "o"),
+                           stitchInfo(name: "m1", incDecValue: 1, symbol: "m")]
+
+let allowedStitches = ["k1", "p1", "ssk", "k2tog", "yo", "m1", "\n"]
```

Comment:
Do you still need this when you have `allowedStitchesInfo`?

Reply:
For the record, you can copy-paste blocks of code into Github comments with the three backticks as well!

That works fine. I would like to suggest that you consider splitting the input on `'\n'` as a first step rather than on `' '`, which would eliminate the need to validate `\n` altogether. But I think that's a bigger lift than is warranted at this stage so I won't consider it a blocker for this PR. But please keep it in mind because splitting on newlines is a very common pattern, and you'll likely have to switch to that anyway when you add in support for reading files.